src/TaskLayout.ts

```diff
@@ -20,6 +21,7 @@ export type TaskLayoutComponent =
     | 'description'
     | 'priority'
     | 'recurrenceRule'
+    | 'createdDate'
```

Comment:
Has this added a new show/hide created date command?

If so, please follow the steps here for adding 'Query parsing' tests of the 2 new commands:
https://publish.obsidian.md/tasks-contributing/Code/How+do+I+add+a+new+task+filter

Reply:
> Has this added a new show/hide created date command?

I saw later that there isn’t a new show/hide command so you can ignite the comment above.

Reply:
Fix the implementation for show/hide command on created date.